core/lib.rs

```diff
@@ -14,7 +14,7 @@ mod errors;
 mod es_isolate;
 mod flags;
 mod module_specifier;
-mod modules;
+pub mod modules;
```

Comment:
I needed to reference `deno_core::modules::Deps` in main.rs
Not sure if there's a better solution

Reply:
Rather than expose the whole module, can you add 
```rust
pub use crate::modules::Deps;
```
We try to maintain a flat namespace.